arrow-core/src/main/kotlin/arrow/core/extensions/option/monadPlus/OptionMonadPlus.kt

```diff
@@ -46,8 +42,8 @@ fun <A> zeroM(): Option<A> = arrow.core.Option
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "plusM(arg1)",
-  "arrow.core.plusM"
+  "orElse { arg1 }",
+  "arrow.core.Option", "arrow.core.orElse"
```

Comment:
```suggestion
  "getOrElse { arg1 }",
  "arrow.core.Option", "arrow.core.getOrElse"
```

Reply:
same `orElse` is the correct method here.